src/demo-app/a11y/dialog/dialog-neptune-iframe-a11y.html

```diff
@@ -0,0 +1,12 @@
+<h2 md-dialog-title>Neptune</h2>
```

Comment:
It does make sense, although specifying what element takes precedence in the focus trap should already be possible through the `cdk-focus-initial` attribute.